### DISCUSSION TITLE
I made a bunch of new sprites for the game, and I want to propose them...

### DISCUSSION THREAD

**Discussion by SecularSteve** [SuperTux/supertux#2020] [Ideas]
I know I'm a little late for the 0.6.3. party, but I want to leave these somewhere where people can decide a bit more easily if they want them for the next release, instead of them getting lost on Discord.

Or maybe even what improvements they can come up with, and suggest themselves.

A "Look Up" sprite (Earth "Shadow" sprite not included because I don't know how to make it):
![look_up_0](https://user-images.githubusercontent.com/33793273/147227135-eb1894e7-19a6-4a96-86cd-fc51b418ba2e.png)![look_up_0](https://user-images.githubusercontent.com/33793273/147227157-3293d8e0-4bd0-4109-acd7-7e4a8e8de8a7.png)![look_up_0](https://user-images.githubusercontent.com/33793273/147227291-b0b32588-6c95-4bd9-b834-6d1b03db16c1.png)![look_up_0](https://user-images.githubusercontent.com/33793273/147227573-e74f7532-6c46-44ba-8f05-5055452be5cc.png)![look_up_0](https://user-images.githubusercontent.com/33793273/147227686-09d16677-2751-4cef-aa53-d5e23e101d64.png)![look_up_0](https://user-images.githubusercontent.com/33793273/147227971-1aefc75a-1b3c-4fc6-9613-baa6a1299acc.png)![look_up_0](https://user-images.githubusercontent.com/33793273/147228156-982cec7c-96e6-435b-8328-33e2b869e0f9.png)![look_up_0](https://user-images.githubusercontent.com/33793273/147228297-2c5eee30-8244-495c-b39e-84b0ce7f2110.png)![look_up_0](https://user-images.githubusercontent.com/33793273/147228434-9c0cd990-6154-4af3-b452-d3dc15e65aef.png)

My proposition for a new Game Over sprite for both comedic effect, and some additional personality to this game:
![gameover-0](https://user-images.githubusercontent.com/33793273/147228600-995eedd1-a333-4591-a756-38b3efda0e9f.png)![gameover-1](https://user-images.githubusercontent.com/33793273/147228602-d28ad961-810f-4098-91c2-16e6c5af886e.png)

My answer to https://github.com/SuperTux/supertux/issues/977 (cycle animations, jumps, kicks, etc. not included):
![stand-hold-0](https://user-images.githubusercontent.com/33793273/147233065-132d4b7e-36fd-4f1c-b39c-9a2f263d8ea2.png)![stand-hold-0](https://user-images.githubusercontent.com/33793273/147233088-90057859-15b5-476a-92c8-b4f8399cf59e.png)

I made them all myself in Krita.
The ZIP contains the .KRA files:
[tux_new_sprites.zip](https://github.com/SuperTux/supertux/files/7768993/tux_new_sprites.zip)

**Comment by Rami-Slicer** [comment]
I like the look up sprites but the game over ones look more weird than comedic.
